README.md

```diff
@@ -16,29 +16,51 @@ You'll also need
 
 If you don't have them, please register yourself and your client from [THETA Developers Website](http://contest.theta360.com/).
 
-## Setup
-* Clone Ricoh Media Storage for Swift and [Ricoh Auth Client for Swift](https://github.com/ricohapi/auth-swift) by running the following command:
+## Dependencies
+* [Ricoh Auth Client for Swift](https://github.com/ricohapi/auth-swift) 1.0+
+
+## Installation
+This section shows you two different methods to install Ricoh Media Storage for Swift in your application.  
+See [Media Storage Sample](https://github.com/ricohapi/media-storage-swift/tree/master/MediaStorageSample#media-storage-sample) to try out a sample of Ricoh Media Storage for Swift.
+
+### CocoaPods
+* If it is your first time to use [CocoaPods](https://cocoapods.org/), run the following commands to set it up.
 ```sh
-$ git clone https://github.com/ricohapi/media-storage-swift.git
-$ git clone https://github.com/ricohapi/auth-swift.git
+$ gem install cocoapods
+$ pod setup
+```
+
+* Go to your project directory.
+* Create a Podfile by running `pod init` ( if you do not have one yet ), and specify `RicohapiMstorage` as follows:
```

Comment:
I thought that the pod name is "RicohAPIMstorage", instead of "RicohapiMstorage". (Or did I miss something...?)